solutions/usaco-809.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: usaco-809
+source: USACO Bronze 2018 Febuary
+title: Taming the Herd
+author: Sathvik Chundru
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_bronze_feb18.html)
+
+<LanguageSection>
+
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class Taming{
+	public static void main(String[] args) throws IOException {
```

Comment:
Either add spacing to both of these before the beginning of the curly brackets or remove them

Reply:
fixed